content/english/chatbot/activity-1.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "Activity 1: Meet ALICE"
+draft: false
+weight: 4
+---
+
+## Meet ALICE
+Before we start building our own chatbot, let's meet ALICE and ask it some questions.

```

Comment:
Potentially provide sample questions to get the user started on types of questions they can ask ALICE.

Reply:
won't fix